### ISSUE TITLE
Any shuffle train dataset?

### ISSUE THREAD

**Issue by 1558359609** [PlusLabNLP/Plot-guided-Coherence-Evaluation#6]
Hi,
It is so beautiful work~! Thanks for your code.
Sorry that I have a question about that I have read the `run_glue.py` code but I haven't found any shuffle for the training dataset.

Besides, I trained your dataset of  `Data/ROC/ROC_Eval/adv_ManPlts/AF_ManPlts_train.tsv` in the roberta model with your origin run_glue code, and the result shows 0.60 spearman and 0.47 kendall correlations.

thanks

**Comment by 1558359609** [comment]
I have found the code about shuffle dataset. 
```
train_sampler = RandomSampler(train_dataset) if args.local_rank == -1 else DistributedSampler(train_dataset)
train_dataloader = DataLoader(train_dataset, sampler=train_sampler, batch_size=args.train_batch_size)

```
